battery

```diff
@@ -75,26 +75,18 @@ battery_charge() {
             fi
             ;;
         "Linux")
-            case $(cat /etc/*-release) in
-                *"Arch Linux"*|*"Ubuntu"*|*"openSUSE"*)
-                    battery_state=$(cat $battery_path/energy_now)
-                    battery_full=$battery_path/energy_full
-                    battery_current=$battery_path/energy_now
-                    ;;
-                *)
-                    battery_state=$(cat $battery_path/status)
-                    battery_full=$battery_path/charge_full
-                    battery_current=$battery_path/charge_now
-                    ;;
-            esac
-            if [ $battery_state == 'Discharging' ]; then
-                BATT_CONNECTED=0
-            else
-                BATT_CONNECTED=1
-            fi
```

Comment:
We do actually rely on this right?

Reply:
Yes but now it defaults to 0 unless one of the batteries is indicating that it is charging or full